packages/runtime/src/templates/getHandler.ts

```diff
@@ -206,6 +206,8 @@ export const getHandler = ({
     throw new Error('Could not find Next.js server')
   }
 
+  process.env.NODE_ENV = 'production';
```

Comment:
What is the reason for this? Just wondering because this will also be the case when running `netlify dev`.

Reply:
We're filtering out `.development.js`, to reduce bundle size:

https://github.com/netlify/next-runtime/blob/main/packages/runtime/src/helpers/functions.ts#L274

(This needs to be done manually for whatever reason)

react-dom is an example for where this helps: https://www.npmjs.com/package/react-dom?activeTab=code

By setting `node_env` to `production`, we ensure these files are never `require`d.

> Just wondering because this will also be the case when running netlify dev.

AFAIK, we're directly proxying `next dev` there - so this runtime wouldn't run.

Reply:
Ah, I see. Thanks, that makes sense. And yes, you're absolutely right about proxying `next dev` - i was having a Friday moment and was mistakenly thinking about `netlify serve`!